### PR TITLE
feat(enhancement): Dev Setup Scripts

### LINKED ISSUE #164: Dev Setup Script
Now that generated files are no longer in the source code, we should create a few commands to enhance the Developer onboarding experience.

```
on another project we've taken the types of source control and using this:
"build:dev": "yarn build:prisma && touch types.ts && yarn build:nexus && yarn codegen", (package.json)

const DEFAULT_BUILD_COMMAND = yarn build:prisma && touch types.ts && yarn build:nexus && yarn codegen && yarn build:next; (buildProd.js)
there is a strange interaction if the types file does not exist prior to running the commands, which is why the touch exists for now
``` 

Thoughts: 
`build:dev` builds the types and files 
`setup:dev`: enhance to leverage `build:dev`, setup the DB, run migrations, seed, etc.

### PR DESCRIPTION
**Pull request by mthomps4**
## Changes 

Dev Setup Scripts:
- Add build:types (prisma, nexus, codgen)
- update buildProd script with new command
- add setup:dev script for generating types and running db:setup

- [x] Generating a new app works

Fixes #164

### CODE REVIEW

**Review by kgajera — CHANGES_REQUESTED** [review]
I think we should also make these changes:

- Remove the `yarn generate` script because it is a duplicate of `yarn build:types`.
- Update the readme and post install instructions to instruct them to use `yarn setup:dev`: 
  - https://github.com/echobind/bisonapp/blob/canary/packages/create-bison-app/postInstallText.ejs#L7
  - https://github.com/echobind/bisonapp/blob/canary/packages/create-bison-app/template/README.md.ejs#L10
- Update the dev server to run `yarn setup:dev`:
  -  https://github.com/echobind/bisonapp/blob/canary/packages/create-bison-app/scripts/createDevAppAndStartServer.js#L34

File: packages/create-bison-app/template/package.json.ejs [hunk 1]
```diff
@@ -30,14 +31,14 @@
...
     "g:test:util": "hygen test util --name",
-    "generate": "yarn build:prisma && yarn build:nexus && yarn codegen",
     "lint": "yarn eslint . --ext .ts,.tsx --fix --ignore-pattern tmp",
     "start": "next start -p $PORT",
     "test": "yarn withEnv:test jest --runInBand --watch",
     "test:ci": "yarn withEnv:test jest --runInBand",
     "test:e2e": "cross-env DOTENV_CONFIG_PATH=.env.test ts-node -r dotenv/config $(yarn bin)/start-server-and-test 'yarn test:server' http://localhost:3001 'yarn cypress:run'",
     "test:e2e:local": "cross-env CYPRESS_LOCAL=true CYPRESS_BASE_URL=http://localhost:3000 cypress open",
     "test:server": "next start --port 3001",
+    "setup:dev": "yarn db:setup && yarn build:types",
```
Comment (kgajera) [inline packages/create-bison-app/template/package.json.ejs:41]: Just checking if we try to order the scripts alphabetically? It seems like they are, at least everything before colons.

Comment (mthomps4) [reply packages/create-bison-app/template/package.json.ejs:41]: A B C D E F G H I J K L M N O P Q R `S`

**Comment by mthomps4** [comment]
Addressed and rebased commits.

**Review by kgajera — APPROVED** [review]

**Pull request merged**